index.html

```diff
@@ -0,0 +1,631 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>New York Times Article</title>
+    <link rel="stylesheet" href="styles.css">
+    <script src="https://kit.fontawesome.com/5b1eeceadd.js" crossorigin="anonymous"></script>
+</head>
+
+<body>
+    <nav class="navbar flex">
+        <div class="left flex">
+            <i class="fas fa-bars"></i>
+            <i class="fas fa-search"></i>
+            <a href="#">SPACE & COSMOS</a>
+        </div>
+        <div class="align">
+            <a href="#"><img class="logo" src="images/New-York-Times-Logo.png" alt="image here"></a>
+        </div>
+        <div class="right flex">
+            <button class="btn">SUBSCRIBE NOW</button>
+            <button class="btn">LOG IN</button>
+        </div>
+    </nav>
+
+    <section class="section-1">
```

Comment:
- [OPTIONAL] Please consider the use of the `<main>` tag for the dominant content of the `<body>` of a document, you can wrap the dominant content with the `<main>` tag